tools/topology/sof/pipe-nr-capture.m4

```diff
@@ -0,0 +1,89 @@
+# Capture Passthrough Pipeline and PCM
```

Comment:
This also to topology commit (I wondered where it is). The name could be perhaps prefixed with "igo" since this is not a generic NR solution, needs the IGO library and license.

Reply:
agree. will change the file name to `pipe-igonr-capture.m4` then.